### PR TITLE
eventing(samples): update Node.js CloudEvents ContainerSource example

### PR DESCRIPTION
**Pull request by lance**
## Proposed Changes 

The example had some old versions of Node and CloudEvents. This change
updates the versions for those dependencies, and makes some modifications
to the example code based on these dependency changes.

Signed-off-by: Lance Ball <lball@redhat.com>

### CODE REVIEW

File: docs/developer/eventing/sources/creating-event-sources/writing-event-source-easy-way/README.md [hunk 1]
```diff
@@ -21,88 +21,101 @@ ContainerSource and SinkBinding both work by injecting environment variables to
...
-The example code uses _Binary_ mode for CloudEvents. To employ structured code, change `let binding = new v1.BinaryHTTPEmitter(config);` to `let binding = new v1.StructuredHTTPEmitter(config);`.
-
-Binary mode is used in most cases because:
-- It is faster in terms of serialization and deserialization.
-- It works better with CloudEvent-aware proxies, such as Knative Channels, and can simply check the header instead of parsing the payload.
-
 ### Procedure
 
+Before publishing the ContainerSource, we must build the application
+image, and push it to a container registry accessible by our cluster.
```
Comment (snneji) [inline docs/developer/eventing/sources/creating-event-sources/writing-event-source-easy-way/README.md:111]:
```suggestion
Before publishing the ContainerSource, you must build the application
image, and push it to a container registry that your cluster can access.
```

**Review by snneji — CHANGES_REQUESTED** [review]

File: docs/developer/eventing/sources/creating-event-sources/writing-event-source-easy-way/README.md [hunk 2]
```diff
@@ -21,88 +21,101 @@ ContainerSource and SinkBinding both work by injecting environment variables to
...
 ```
 
-**NOTE:** Due to this [bug](https://github.com/cloudevents/sdk-javascript/issues/191), you must use version 2.0.1 of the Javascript SDK or newer.
-
 ## Using ContainerSource
 
 A ContainerSource creates a container for your event source image and manages this container.
 
-The sink URL to post the events will be made available to the application through the `K_SINK` environment variable by the ContainerSource.
+The sink URL where events are posted will be made available to the application through the `K_SINK` environment variable by the ContainerSource.
```
Comment (abrennan89) [inline docs/developer/eventing/sources/creating-event-sources/writing-event-source-easy-way/README.md:32]:
```suggestion
The sink URL where events are posted is made available to the application through the `K_SINK` environment variable, provided by the ContainerSource.
```

**Review by abrennan89 — APPROVED** [review]
/lgtm

One minor suggestion @lance but I don't think it's worth blocking this PR over. Thanks!

**Pull request merged**